enqueue/amqp-ext/0.3/etc/packages/enqueue_amqp.yaml

```diff
@@ -0,0 +1,23 @@
+enqueue:
+    transport:
+        default: 'amqp'
+        amqp:
+            host: '%env(ENQUEUE_AMQP_HOST)%'
+            port: '%env(ENQUEUE_AMQP_PORT)%'
+            login: '%env(ENQUEUE_AMQP_LOGIN)%'
+            password: '%env(ENQUEUE_AMQP_PASSWORD)%'
+            vhost: '%env(ENQUEUE_AMQP_VHOST)%'
```

Comment:
I've have had a look, but do you support a URL/DSN as well? That's what we are using for Doctrine and Swiftmailer, so that would be great if possible to do the same here as well.